apps/docs/content/guides/platform/manage-your-usage.mdx

```diff
@@ -0,0 +1,4 @@
+---
+id: 'manage-your-usage'
+title: 'Manage your usage'
+---
```

Comment:
We'll need some kind of text here ,even if it's only a link to subpages, it looks too weird without any content

Reply:
Yes. Either the link, kind of an explanation text what the usage pages are about and/or info that is relevant for all the subpages. E.g. "what does 'GB-Hrs' mean?"

Gonna add links for now.